docs/source/en/_toctree.yml

```
@@ -165,6 +165,8 @@
         title: Self-Attention Guidance
       - local: api/pipelines/stable_diffusion/panorama
         title: MultiDiffusion Panorama
+      - local: api/pipelines/stable_diffusion/control_net
```

```suggestion
      - local: api/pipelines/stable_diffusion/controlnet
```

Maybe rename the file to have a simpler URL?

---

I think we could also add a small guide (maybe with links to this documentation) in the "Pipelines for Inference" section, and a reference from the "Controlling generation" page.

---

> Maybe rename the file to have a simpler URL?

Yes, that's better. This one has been corrected as well. 
https://github.com/huggingface/diffusers/pull/2407/commits/3f6e8f7aef1adc44883044e77454d7f64820bc01
https://github.com/huggingface/diffusers/pull/2407/commits/ebabcbec6c0f8ad6a08237de0be96627d2666e52

---

> I think we could also add a small guide (maybe with links to this documentation) in the "Pipelines for Inference" section, and a reference from the "Controlling generation" page.


I didn't know this page existed. 

https://github.com/huggingface/diffusers/pull/2407/commits/30e2bde64f0636848b02d60001b780f3876eb50a

It's a copy and paste of the abstract part of the pipeline page. Does it work as intended?